### ISSUE TITLE
a couple hivemind bugs

### ISSUE THREAD

**Issue by Suicidalpickles** [tgstation/tgstation#43001]
## Round ID: 
#102967

## Testmerges:
#42930 

## Reproduction:
1) Ok so basically, after being blown up by another hivemind and having your hosts stolen; if you are cloned, you still keep all of your powers but they look greyed out UI-wise, yet these buttons still function. Not sure if that part is intentional.

2) Hivemind Circadian Shift seems to have some weird things going on with it too. (AKA the spell thats an AOE sleep.) Basically, I had some chasing and pinpointing me with their mind tracker and I clicked the spell. It sleeped a nearby afk assistant in range, yet it didn't put the dude chasing after me to sleep, who was like 2 tiles away. Also as extra info, they were a hivemind too but they managed to put me to sleep just fine from the same distance.

EDIT: Just read the original hivemind 2.0 PR and saw 
>"Hivemind hosts (but not mind controlled vessels) are immune to this power, a saving grace for anybody not wanting to instantly die to Circadian into Reclaim."

tl;dr somehow got circadian'd by another hivemind even though they're supposed to be immune to it.

**Comment by Kierany9** [comment]
Number 1 isn't intentional and I'll look into it soon:tm:
As for Number 2, if he was standing still to KO you it may have been Neural Shock. Otherwise I have no idea what's going on there.

**Comment by Kierany9** [comment]
Number 1 has been fixed
